selenium/backend/src/test/scala/io/udash/selenium/SeleniumTest.scala

```diff
@@ -0,0 +1,35 @@
+package io.udash.selenium
+
+import java.util.concurrent.TimeUnit
+
+import io.udash.selenium.server.ApplicationServer
+import org.openqa.selenium.Dimension
+import org.openqa.selenium.firefox.{FirefoxDriver, FirefoxOptions}
+import org.openqa.selenium.remote.RemoteWebDriver
+import org.scalatest.concurrent.Eventually
+import org.scalatest.time.{Millis, Seconds, Span}
+import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpec}
+
+abstract class SeleniumTest extends WordSpec with Matchers with BeforeAndAfterAll with Eventually {
+  override implicit val patienceConfig: PatienceConfig =
+    PatienceConfig(scaled(Span(10, Seconds)), scaled(Span(5, Millis)))
+
+//  val driver: RemoteWebDriver = new ChromeDriver(new ChromeOptions().setHeadless(false))
+  val driver: RemoteWebDriver = new FirefoxDriver(new FirefoxOptions().setHeadless(true))
```

Comment:
The driver and the server should be started in `beforeAll()` because if the constructor fails, `afterAll()` can't be called. So for example if the server fails to start (e.g. some port is already used) then the browser and driver are not closed.